### PR TITLE
Minor improvements 

### PR DESCRIPTION
**Pull request by katjanakosic**
I added some minor improvements to the documentation like spelling and formatting. Also did some proofread.

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "esbonio.sphinx.confDir": ""
+}
```
Comment (pal03377) [inline .vscode/settings.json]: VS Code also kept adding this for me, but can you remove this? I don't think it's really relevant

**Review by pal03377 — COMMENTED** [review]
At first glance, the changes look really good! I'm going to wait until all the painful conflicts are solved before approving though 😅

File: docs/introduction/purpose.rst [hunk 2]
```diff
@@ -9,12 +9,12 @@ an open-source learning management system used at universities, Themis allows tu
 inspect students' code and provide feedback on submissions.
 
 Designed as a native iPad application, Themis optimizes viewing and assessing programming exercises on a mobile
-device. Its intuitive interface and user-friendly assessment editor enable tutors to give both general as well as
+device. Its' intuitive interface and user-friendly assessment editor enable tutors to give both general as well as
```
Comment (huber-florian) [inline docs/introduction/purpose.rst:12]: It is not "Its'" as far as I know. "Its" was correct.

**Comment by huber-florian** [comment]
Apart from that I also wait with reviewing until you have resolved the conflicts.

File: docs/admin/third-party.rst [hunk 3]
```diff
@@ -0,0 +1,100 @@
+Third-party Components
```
Comment (huber-florian) [inline docs/admin/third-party.rst]: You probably have to copy this whole file into its new place or delete it depending on if you did any changes to it.

Comment (katjanakosic) [reply docs/admin/third-party.rst]: Done

File: docs/introduction/purpose.rst [hunk 4]
```diff
@@ -9,12 +9,12 @@ an open-source learning management system used at universities, Themis allows tu
 inspect students' code and provide feedback on submissions.
 
 Designed as a native iPad application, Themis optimizes viewing and assessing programming exercises on a mobile
-device. Its intuitive interface and user-friendly assessment editor enable tutors to give both general as well as
```
Comment (crustypluto19) [inline docs/introduction/purpose.rst:12]: No need to change it, it was correct without the apostrophe :)

**Review by crustypluto19 — APPROVED** [review]
Minor grammatical error, otherwise looks good!
I'm still gonna approve it to speed things up.

**Comment by pal03377** [comment]
I still didn't get to review the contents, unfortunately, but I did do a quick pull and these 3 new warnings appear when building:
```
/.../Themis/docs/admin/themisml/deployment.rst:2: WARNING: Explicit markup ends without a blank line; unexpected unindent.
/.../Themis/docs/analysis/ui.rst:5: WARNING: Explicit markup ends without a blank line; unexpected unindent.
/.../Themis/docs/system/access-control.rst:2: WARNING: Explicit markup ends without a blank line; unexpected unindent.
```

Could you please look into those?

File: docs/admin/themisml/deployment.rst [hunk 5]
```diff
@@ -31,7 +31,7 @@ redirects all incoming traffic from port 80 to port 443 to ensure a secured conn
 uses a SSL/TLS certificate from the TUM Rechnerbetriebsgruppe.
 
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Manual restart of the ThemisML server:
+Manual restart of the ThemisML Server:
```
Comment (pal03377) [inline docs/admin/themisml/deployment.rst]: Maybe capitalize restart as well?
```suggestion
Manual Restart of the ThemisML Server:
```

File: docs/introduction/purpose.rst (hunk 2 above)
Comment (pal03377) [reply docs/introduction/purpose.rst:12]: Is this already fixed? I still see the incorrect `Its'`, but I also sometimes have trouble with GitHub not updating files I think 🤔

**Review by pal03377 — APPROVED** [review]
Thank you for the thorough review! Very nice 🥳

**Comment by pal03377** [comment]
Turns out, the warnings were coming from changes in my own PR 🙈 

They are now also fixed, I think we can merge now! 🎉

**Pull request merged**